src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="row">
-  <div class="col-lg-3 col-sm-6 col-xs-12">
+  <div class="col-lg-5 col-md-12 col-sm-6 col-xs-12">
```

Comment:
Such a column size combination was chosen due to the column size combination chosen for the recipient name, i.e. `col-md-5 col-xs-12`, in `question-submission-form.component.html`. When the screen size is `sm`, the recipient name will take up 2 columns and it looks nicer if the dropdown list and the more info link each take up 1 column, as compared to both taking up 2 columns.

For your reference.
Column size combination: `col-lg-5 col-md-12 col-xs-12`
![image](https://user-images.githubusercontent.com/75534016/176363351-b8f1573f-ae95-4b83-8e95-fa7928a8dbbd.png)

Column size combination: `col-lg-5 col-md-12 col-sm-6 col-xs-12`
![image](https://user-images.githubusercontent.com/75534016/176363536-51dc0d76-e837-4a86-846c-16a9dfc40cdb.png)

What do you think?

Reply:
Agree that it looks nicer on smaller screens. My original comment should have been this (apologies, another typo).

md-lg: 1 columns,
sm-md: 2 column,
xs-sm: back to 1 columns

Seems odd that sm-md is suddenly 2 columns. Guessing the intention was for it to be one column below a certain screen size.

Reply:
> When the screen size is sm, the recipient name will take up 2 columns and it looks nicer if the dropdown list and the more info link each take up 1 column, as compared to both taking up 2 columns.

Was this not implemented? Seems like this was marked as resolved without any resolution. Or did you mean to say rows instead of columns here?

Reply:
Oh wait, before that, we are going with the second screenshot above right? If so, the current code is already implementing that.

Reply:
Nvm, we can leave as this.